backend/src/gh.rs

```diff
@@ -99,12 +99,12 @@ struct AccessTokenResponse {
 /// Request a github installation access token using the provided reqwest client.
 /// The installation access token will expire after 1 hour.
 /// Returns the new token, and the time of expiration
-async fn get_access_token(req_client: &Client) -> Result<(String, SystemTime)> {
-    let token = gen_jwt_token()?;
+async fn get_access_token(req_client: &Client, config: Arc<HydeConfig>) -> Result<(String, SystemTime)> {
+    let token = gen_jwt_token(Arc::clone(&config))?;
```

Comment:
Doing that would also mean that you dno't need to clone an arc here, just pass in an `&str`